perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -31,11 +31,6 @@ sub restriction {
     return { cobrand => shift->moniker };
 }
 
-sub problems_restriction {
-    my ($self, $rs) = @_;
-    return $rs->to_body($self->council_id);
-}
```

Comment:
The way you suggest is what I'd originally done, but broke it out into `admin_problems`/`problems` after we chatted about it. Would a HTTP 301 for non-owned reports redirecting to FMS.com be OK here?